SegmentEditorMaskVolume/SegmentEditorMaskVolumeLib/SegmentEditorEffect.py

```diff
@@ -44,19 +45,58 @@ def setupOptionsFrame(self):
     self.operationRadioButtons.append(self.fillOutsideButton)
     self.buttonToOperationNameMap[self.fillOutsideButton] = 'FILL_OUTSIDE'
 
+    self.binaryMaskFillButton = qt.QRadioButton("Binary mask")
```

Comment:
@lassoan I asked a few people and it does seem as though it may not be as obvious a name as it could be. I'm not sure what might be more explicit yet still simple enough to have as an operation name; something that describes the fill inside/outside functionality but also that a labelmap volume is being generated.